Samples/HelloWorld/OrleansHost/OrleansHostWrapper.cs

```diff
@@ -35,31 +34,30 @@ public int Run()
             if (siloHost == null)
             {
                 Console.Error.WriteLine("Failed to initialize Orleans silo due to bad command line arguments");
+                SiloArgs.PrintUsage();
                 return 1;
             }
-            else
-                try
-                {
-                    siloHost.InitializeOrleansSilo();
 
-                    if (siloHost.StartOrleansSilo())
-                    {
-                        Console.WriteLine("Successfully started Orleans silo '{0}' as a {1} node.", siloHost.Name,
-                            siloHost.Type);
-                        return 0;
-                    }
-                    else
-                    {
-                        throw new OrleansException(string.Format("Failed to start Orleans silo '{0}' as a {1} node.",
-                            siloHost.Name, siloHost.Type));
-                    }
+            try
+            {
+                siloHost.InitializeOrleansSilo();
+
+                if (siloHost.StartOrleansSilo())
+                {
+                    Console.WriteLine($"Successfully started Orleans silo '{siloHost.Name}' as a {siloHost.Type} node.");
+                    return 0;
                 }
-                catch (Exception exc)
+                else
                 {
-                    siloHost.ReportStartupError(exc);
-                    Console.Error.WriteLine(exc);
-                    return 1;
+                    throw new OrleansException($"Failed to start Orleans silo '{siloHost.Name}' as a {siloHost.Type} node.");
                 }
+            }
+            catch (Exception exc)
+            {
+                siloHost.ReportStartupError(exc);
+                Console.Error.WriteLine(exc);
```

Comment:
should we be consistent on whether we print exception to console or a log file, because some place we print it to console, someplace we print it to a log. As a small program, we might just print it on console.